File: article/src/main/java/com/ruzhan/awaker/article/model/NewEle.java
package com.ruzhan.awaker.article.model;

/**
 * Created by ruzhan on 2017/7/15.
 */

public class NewEle {

    public static final String TAG_PERIOD = "。";
    public static final String TAG_LOGO = "觉醒字幕组";

    public static final String TAG_P = "p";
    public static final String TAG_IFRAME = "iframe";
    public static final String TAG_EMBED = "embed";
    public static final String TAG_STRONG = "strong";

    public static final String IMG = "img";
    public static final String SRC = "src";

    public static final int TYPE_TEXT = 1000;
    public static final int TYPE_IMG = 1001;
    public static final int TYPE_VIDEO = 1002;

    public int type;
    public String text;
    public boolean isBold;
    public String imgUrl;
    public String videoUrl;
    public String html;
}

File: article/src/main/java/com/ruzhan/awaker/article/model/NewEle.kt
package com.ruzhan.awaker.article.model

class NewEle {

    var type: Int = 0
    var text: String? = null
    var isBold: Boolean = false
    var imgUrl: String? = null
    var videoUrl: String? = null
    var html: String? = null

    companion object {

        val TAG_PERIOD = "。"
        val TAG_LOGO = "觉醒字幕组"

        val TAG_P = "p"
        val TAG_IFRAME = "iframe"
        val TAG_EMBED = "embed"
        val TAG_STRONG = "strong"

        val IMG = "img"
        val SRC = "src"

        val TYPE_TEXT = 1000
        val TYPE_IMG = 1001
        val TYPE_VIDEO = 1002
    }
}
